core/src/main/java/bisq/core/offer/OfferPayload.java

```diff
@@ -51,7 +51,7 @@
 @EqualsAndHashCode
 @Getter
 @Slf4j
-public final class OfferPayload implements ProtectedStoragePayload, ExpirablePayload, RequiresOwnerIsOnlinePayload {
+public class OfferPayload implements ProtectedStoragePayload, ExpirablePayload, RequiresOwnerIsOnlinePayload {
```

Comment:
Should stay final for security reasons

Reply:
it is final (not in this commit but later)

Reply:
Ah saw it now... thanks.